baseplate/frameworks/queue_consumer/kombu.py

```diff
@@ -182,8 +195,11 @@ def new(
         :param queue_name: Name for your queue.
         :param routing_keys: List of routing keys that you will create
             :py:class:`~kombu.Queue` s to consume from.
-        :param handler_fn: A `baseplate.frameworks.queue_consumer.komub.Handler`
+        :param handler_fn: A `baseplate.frameworks.queue_consumer.kombu.Handler`
```

Comment:
:grimacing: thanks

Reply:
actually, in both of these: is it actually necessary to spell out the type like this? it should be auto-linked by the type hint in the sphinx docs.

Reply:
Good point, I did not know how these comments are parsed out. I've updated it as suggested